tests/integration/components/hyper-table-v2-test.ts

```diff
@@ -79,6 +80,18 @@ module('Integration | Component | hyper-table-v2', function (hooks) {
     assert.ok(handlerSpy.triggerEvent.calledOnceWithExactly('row-click', this.handler.rows[0]));
   });
 
+  test('when destroyed, the table properly calls the handler teardown method', async function (this: TestContext, assert: Assert) {
+    this.displayTable = true;
+
+    const teardownStub = sinon.stub(this.handler, 'teardown');
+
+    await render(hbs`{{#if this.displayTable}}<HyperTableV2 @handler={{this.handler}} />{{/if}}`);
+
+    this.set('displayTable', false);
+
+    assert.ok(teardownStub.calledOnce);
```

Comment:
Nit: could you group these lines more ?

If you want more space between your lines, you can configure your IDE :trollface: 

A break between setup and assertions is fine IMO, more is not justified.